README.md

```diff
@@ -0,0 +1,44 @@
+# Welcome to PubDoc
+
+## TL;DR
+The goal of this repository is twofold:
+1. To make documentation better for the reader.
+2. To make _creating_ documentation a better experience for the writer.
+
+## How to Use
+This repository is currently very "bare bones." To run the tool and interface with Confluence,  
+you may need to contact an IT administrator to obtain permissions to access Confluence data.  
+
+Visit [this link](https://id.atlassian.com/manage-profile/security/api-tokens) to create your API token.  
+Next, create a `.env` file with the following contents:
+
+### `.env` File Example
+```env
+CONFLUENCE_USERNAME=<username>
+CONFLUENCE_API_KEY=<api key>
+CONFLUENCE_DOMAIN=<your company>.atlassian.net
+CONFLUENCE_SPACEID=<A space you don't care about littering in>
+```
+To get a spaceID of a space. Go to the space then click on "space settings" in the top left. Then run this in the browser to get the spaceID. `https://YOURURL.atlassian.net/wiki/rest/api/space/KEY`
+
+## Pontifications & Justifications
```

Comment:
Love this section lol. Although since I made this a public repo under my account rather than a private repo in the `flexgen` organization, I purposefully left it anonymous and without FlexGen-specific stuff. We could go ahead and move all of this into a branch in our monorepo if we want to include FlexGen-specific stuff

Reply:
We'd also need to get the `blackfriday` dependency reviewed & approved. Even the name itself, "blackfriday," sounds like the name of the latest malware attack you hear about in the news

Reply:
FWIW I have already noticed undesirable formatting in the generated html using that package. 
You can pubdoc the readme if you want to see it. The numbered list gets janked. 

The only reference to flexgen I see in the README.md is me referring back to iframe work I did for graphing behavior of SC in confluence. I think that's good work, but I don't think there is appetite for that to be supported atm. It's a good idea though. 

I think at the end of the day that'll get jammed into <flexgen ip thing you do> at some point.

Reply:
![image](https://github.com/user-attachments/assets/87b1d631-f76c-4cc4-831a-40d623513d26)

Reply:
I noticed when I tried using `-`s to do a bulleted list, I had to put spaces in between the `-` lines in order for the rendered HTML to understand it as a bulleted list. I suspect the same thing is the case for the numbered list?

Reply:
It also seems like it wants you to just keep things on the same line if you don't want them on separate lines in the HTML

Reply:
Totally open to exploring other Markdown -> HTML converters

Reply:
If we kept this app generic, I wonder if it could become a cool thing to show FlexGen supports some open source work 🤔

Reply:
Cool!!